deployment-guide.md

```diff
@@ -16,7 +16,191 @@ the central services.
 
 ### FAIR Data Point
 
-https://docs.fairdatapoint.org/en/latest/deployment/local-deployment.html
+#### Installation
+
+First, create a docker compose file (e.g. `compose.yml`) as this in an empty directory:
+
+```yaml
+ # compose.yml
+
+services:
+  fdp:
+    image: "fairdata/fairdatapoint:${FDP_VERSION:-1.18}"
+    restart: no
+    environment:
+      SERVER_PORT: 8080
+      INSTANCE_CLIENTURL: http://localhost
+      INSTANCE_PERSISTENTURL: http://localhost
+    depends_on:
+      mongo:
+        condition: service_healthy
+      graphdb:
+        condition: service_healthy
+    healthcheck:
+      test: wget --quiet --spider http://127.0.0.1:8080 || exit 1
+      start_interval: 3s
+      start_period: 30s
+
+  fdp-client:
+      image: "fairdata/fairdatapoint-client:${FDP_CLIENT_VERSION:-1.18}"
+      restart: no
+      ports:
+        - "127.0.0.1:80:80"
+      environment:
+        FDP_HOST: fdp:8080
+      depends_on:
+        fdp:
+          condition: service_healthy
+      healthcheck:
+        test: wget --quiet --spider http://127.0.0.1 || exit 1
+        start_interval: 3s
+        start_period: 30s
+
+  mongo:
+    image: "mongo:${MONGO_VERSION:-8.0}"
+    restart: no
+    healthcheck:
+      test: |
+        [ $(mongosh --quiet --host localhost:27017 --eval "db.runCommand('ping').ok") = 1 ] || exit 1
+      start_interval: 3s
+      start_period: 30s
+
+  graphdb:
+    image: ontotext/graphdb:10.7.6
+    ports:
+        - 7200:7200
```

Comment:
`fdp-client` is restricted to exposing its port to localhost, but graphdb isn't. It's probably good to be consistent and restrict both, assuming they will be served behind a reverse proxy.